app/assets/javascripts/search_ideas.js

```diff
@@ -0,0 +1,20 @@
+var searchForIdea = function(){
+  $('#search').on('keyup', function(e){
+    e.preventDefault()
+
+    var query = $(this).val()
+
+    var ideas = $('.idea');
+
+    $.each(ideas, function(index, idea){
+      var title = $(idea).find('.title').text()
+      var body = $(idea).find('.body').text()
+
+      if (title.includes(query) || body.includes(query)) {
```

Comment:
This could be refactored to;

```js
ideas.each(function (index, idea) {
  var title = $(idea).find('.title').text()
  var body = $(idea).find('.body').text()

   var matchesSearchTerm = title.includes(query) || body.includes(query)

  $(idea).toggle(matchesSearchTerm)
})
```

Also, `includes` is new and doesn't have great support. It works in modern Chrome, but doesn't work in a bunch of not-so-old browser.